### PR TITLE
feat(tests): uses new @web/test-runner instead of karma #27

### PR DESCRIPTION
**Pull request by quarkus**

### CODE REVIEW

**Comment by eddyloewen** [comment]
@quarkus What about the karma.conf.js file? Is it still needed?

**Comment by quarkus** [comment]
> @quarkus What about the karma.conf.js file? Is it still needed?

ope. removed

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 dist/
 node_modules/
 test/coverage
+coverage
```
Comment (eddyloewen) [inline .gitignore:4]: @quarkus can we also configure the test coverage to be placed in test/coverage as before?

Comment (quarkus) [reply .gitignore]: check

**Review by eddyloewen — APPROVED** [review]

**Comment by eddyloewen** [comment]
Nice! But what is it with all the white space changes?! I don't get it! That was supposed to NOT happen...
Has something changed?! Do we not use the git-hooks anymore?

**Comment by quarkus** [comment]
> Nice! But what is it with all the white space changes?! I don't get it! That was supposed to NOT happen... Has something changed?! Do we not use the git-hooks anymore?

I think it was actually broken (not well formatted) before and got fixed now (by prettier / husky) cause i had to touch all the test files.

But i am pretty sure that our prettier config is broken as it inserts tabs instead of spaces ... and tabs is false by default.
I´ll raise an issue for that ..

**Pull request merged**